### ISSUE TITLE
divide-by-zero in jpc_dec_process_siz (jpc_dec.c) (DIFFERENT FROM #22)

### ISSUE THREAD

**Issue by asarubbo** [mdadams/jasper#23]
Hello,
time ago I found multiple crashes on jasper. I didn't know where to post the bugs since the development seems dead, so I just informed the community on oss-security.
Now I discovered that the development is still active, here the details:

```
# imginfo $FILE
warning: trailing garbage in marker segment (5 bytes)
ASAN:DEADLYSIGNAL
=================================================================
==24077==ERROR: AddressSanitizer: FPE on unknown address 0x7f78c36f9822 (pc 0x7f78c36f9822 bp 0x7ffe2bff10c0 sp 0x7ffe2bff0fa0 T0)
    #0 0x7f78c36f9821 in jpc_dec_process_siz /tmp/jasper-version-1.900.3/src/libjasper/jpc/jpc_dec.c:1196:18
    #1 0x7f78c36ff2b2 in jpc_dec_decode /tmp/jasper-version-1.900.3/src/libjasper/jpc/jpc_dec.c:390:10
    #2 0x7f78c36ff2b2 in jpc_decode /tmp/jasper-version-1.900.3/src/libjasper/jpc/jpc_dec.c:254
    #3 0x7f78c36a2a9a in jas_image_decode /tmp/jasper-version-1.900.3/src/libjasper/base/jas_image.c:372:16
    #4 0x4f11bd in main /tmp/jasper-version-1.900.3/src/appl/imginfo.c:179:16
    #5 0x7f78c27ba61f in __libc_start_main /var/tmp/portage/sys-libs/glibc-2.22-r4/work/glibc-2.22/csu/libc-start.c:289
    #6 0x418bc8 in _start (/tmp/jasper-version-1.900.3/src/appl/.libs/imginfo+0x418bc8)

AddressSanitizer can not provide additional info.
SUMMARY: AddressSanitizer: FPE /tmp/jasper-version-1.900.3/src/libjasper/jpc/jpc_dec.c:1196:18 in jpc_dec_process_siz
==24077==ABORTING
```

Tested against the latest 1.900.3
Testcase:
[12.crash.zip](https://github.com/mdadams/jasper/files/527238/12.crash.zip)

**Comment by mdadams** [comment]
This problem is also fixed on the master branch now.  Same problem as in 11.crash.zip.

**Comment by asarubbo** [comment]
Could you link the exactly commit which fixes the issue (if different from #22) ? thanks.

**Comment by mdadams** [comment]
The same commit fixed both.  One of your test cases was failing due to a problem with a bad XRsiz field and one was due to a bad YRsiz field.  The commit that fixed #22  added a check for both fields and therefore fixed the bugs for both test cases.
